pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -21,14 +21,22 @@ eexpect ":/# "
 end_test
 
 start_test "Check that a broken stderr prints a message to the log files."
-send "$argv start-single-node -s=path=logs/db --insecure --logtostderr --vmodule=*=1 2>&1 | cat\r"
+# We use --log-file-max-size to avoid rotations during the test.
+send "$argv start-single-node -s=path=logs/db --log-file-max-size=100M --insecure --logtostderr --vmodule=*=1 2>&1 | cat\r"
 eexpect "CockroachDB node starting"
 system "killall cat"
 eexpect ":/# "
 # NB: we can't just grep for the broken pipe output, because it may take
 # a while for the server to initiate the next log line where it will detect
 # the broken pipe error.
-send "tail -f logs/db/logs/cockroach.log\r"
+#
+# We use -F and not -f because the log file may be rotated during the test.
+#
+# We also watch both the main log file and the pebble log in the
+# directory, because the error is only reported on the logger which is
+# writing first after stderr is has been broken, and that may be the
+# secondary logger.
+send "tail -F logs/db/logs/cockroach{,-pebble}.log\r"
```

Comment:
This looks like it could rot again, is there a downside to globbing all .log files?

Reply:
Good idea, i'll do that.